### ISSUE TITLE
Forum

### ISSUE THREAD

**Issue by JuKu** [JuKu/JuKuCMS#211]
Should we install a forum, so others can interact with us, write questions and to build up a community?

**Comment by JuKu** [comment]
@PascalReintjens

**Comment by PascalReintjens** [comment]
Yes, this would be great, but I think we shouldn't invest too much time for that at this state of development

**Comment by JuKu** [comment]
@PascalReintjens My plan for future is to get more contributors and really important plugin developers, because most features has to be implemented as plugins (so user can replace / extend system with plugins).
But you are right.
